libraries/Microsoft.Bot.Connector/Authentication/AppCredentials.cs

```diff
@@ -146,6 +146,7 @@ public string ChannelAuthTenant
         /// </summary>
         /// <param name="serviceUrl">The service URL.</param>
         /// <remarks>If expiration time is not provided, the expiration time will DateTime.UtcNow.AddDays(1).</remarks>
+        [Obsolete("TrustServiceUrl is not a required part of the security model.")]
```

Comment:
Should we also add a deprecation message around IsTrustedServiceUrl(string serviceUrl)

Reply:
so annoyed I missed that! thanks for spotting it

Reply:
and yes about the doc issue

Reply:
and I just found a test case to drop too